### ISSUE TITLE
runtime error createEvent

### ISSUE THREAD

**Issue by ik1001** [iodigital-com/SecretSanta#573]
using Google chrome i could not create an event. 
i get error cannot read property of null (reading 'value') on the line in the picture.

prevents the validation email from being sent out and  user get redirected to what i assume is the spam landing page.
![error](https://github.com/iodigital-com/SecretSanta/assets/12459662/32b58dd7-c194-469a-93f4-e982ab6c0ce8)

**Comment by tvlooy** [comment]
Where exactly did you get redirected to?

**Comment by ik1001** [comment]
seems to happen when i add more than 8 people
https://www.secretsantaorganizer.com/party/create
![image](https://github.com/iodigital-com/SecretSanta/assets/12459662/50ee7219-aaba-48d4-856f-e3a3da5fa135)

**Comment by tvlooy** [comment]
Are you behind a proxy that multiple people use or something? I can't reproduce at the moment

**Comment by ik1001** [comment]
no, tried it from incognito mode too and same issue, i was able to create my event by disabling the code in the picture. 
i think it might be because one of the emails look like this  example_@gmail.com

**Comment by tvlooy** [comment]
I think it happens because rate limiting is checked before form validation. I deployed a fix https://github.com/iodigital-com/SecretSanta/commit/42837a7248635d50309c2080c3e68e0939b8da25 for this. Could you try again?
